server/api/router.go

```diff
@@ -220,6 +220,10 @@ func createRouter(ctx context.Context, prefix string, svr *server.Server) *mux.R
 	apiRouter.Handle("/metric/query", newQueryMetric(svr)).Methods("GET", "POST")
 	apiRouter.Handle("/metric/query_range", newQueryMetric(svr)).Methods("GET", "POST")
 
+	// tso API
+	tsoHandler := newTsoHandler(svr, rd)
+	apiRouter.HandleFunc("/tso/local/transfer/{name}", tsoHandler.TransferLocalTSOAllocator).Methods("POST")
```

Comment:
What about using `/tso/allocator/transfer/{name}` instead? It could make the purpose of this API more clear.